setup.py

```diff
@@ -203,7 +262,7 @@ def get_exts_for(name):
         "Programming Language :: Python :: 3.10",
     ],
     use_scm_version={"write_to": "ddtrace/_version.py"},
-    setup_requires=["setuptools_scm[toml]>=4,<6.1", "cython"],
+    setup_requires=["setuptools_scm[toml]>=4,<6.1", "cython", "cmake", "ninja"],
```

Comment:
Should we use cmake from the OS rather than pip?

Reply:
No strong opinion, it looks up-to-date on PYPI and will make most installation from source succeed without installing extra dependencies (apart from g++ & git). Using cmake from the OS works too but is a bit more a hassle on Windows for example.

Reply:
@jd I guess the impact is the install size? I have no strong opinion either, so I can try to remove

Reply:
I might be in favor of using the `pip` version of `cmake` myself, because while we control the build environment..... most of the time.... having it here is probably helpful for any customers that do have to build from source.

Reply:
.... and windows.

Reply:
Let's use the pip one for now then